pages/watches/catfish.md

```diff
@@ -5,3 +5,8 @@ label: catfish/catfish_ext/catshark
 section: install
 layout: aw-install
 ---
+<div class="callout callout-info">
+    <p>Hardware support notes:</p>
+    <p>Dual-Display support<br> Is marked partial due to incomplete support. The secondary LCD works on all models and shows the time and steps when enabled. However, enabling the step counter and the timepiece mode is currently not available through the user interface.</p>
```

Comment:
I'd add '... shows time and steps when enabled from cli.' just to hint at how it's possible to enable it for users who aren't aware.
otherwise, looking good!

Reply:
possibly add with what commands and switches? Something like: "You can enable step counting with 'lcd-tools --enable-stepcounter'"

Reply:
> possibly add with what commands and switches? Something like: "You can enable step counting with `lcd-tools --enable-stepcounter`"

I think that is out of scope of the install pages and belongs to the affected watches wiki entry?

Reply:
I think a adding such information to a wiki page is indeed better.